src/utils.ts

```diff
@@ -44,7 +44,7 @@ export function objectify(arg: any): any {
  * E.g. "foo-bar" -> "fooBar"
  */
 export function camelize(stringWithHyphens: string): string {
-    return stringWithHyphens.replace(/-(.)/g, (_, nextLetter) => nextLetter.toUpperCase());
+    return stringWithHyphens.replace(/-(.)/g, (_, nextLetter) => (nextLetter as string).toUpperCase());
```

Comment:
I don't like that style since it behaves the same as a typecast but doesn't look like one. Was actually considering writing a lint rule for that.